### PR TITLE
[Documentation][API] Fix not working curl commands 

### PR DESCRIPTION
**Pull request by tuka217**
| Q               | A
| --------------- | ---
| Bug fix?        | no|
| New feature?    | no|
| BC breaks?      | no|
| Related tickets | -|
| License         | MIT |

### CODE REVIEW

File: docs/api/customers.rst [hunk 1]
```diff
@@ -60,6 +60,195 @@ If you request for a more detailed data, you will receive an object with followi
...
++--------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| Parameter                | Parameter type | Description                                                                                          |
++==========================+================+======================================================================================================+
+| Authorization            | header         | Token received during authentication                                                                 |
++--------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| email                    | request        | **(unique)** Customers email                                                                         |
++--------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| firstName                | request        | Customers first name                                                                                 |
```
Comment (TheMadeleine) [inline docs/api/customers.rst]: Customer's

File: docs/api/customers.rst [hunk 2]
```diff
@@ -60,6 +60,195 @@ If you request for a more detailed data, you will receive an object with followi
...
+.. code-block:: text
+
+    POST /api/v1/customers/
+
++--------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| Parameter                | Parameter type | Description                                                                                          |
++==========================+================+======================================================================================================+
+| Authorization            | header         | Token received during authentication                                                                 |
++--------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| email                    | request        | **(unique)** Customers email                                                                         |
```
Comment (TheMadeleine) [inline docs/api/customers.rst]: Customer's

File: docs/api/customers.rst [hunk 3]
```diff
@@ -60,6 +60,195 @@ If you request for a more detailed data, you will receive an object with followi
...
++--------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| firstName                | request        | Customers first name                                                                                 |
++--------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| lastName                 | request        | Customers last name                                                                                  |
```
Comment (TheMadeleine) [inline docs/api/customers.rst]: Customer's

File: docs/api/customers.rst [hunk 4]
```diff
@@ -60,6 +60,195 @@ If you request for a more detailed data, you will receive an object with followi
...
++--------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| groups                   | request        | *(optional)* Array of groups customer belongs to                                                     |
++--------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| gender                   | request        | Customers gender                                                                                     |
```
Comment (TheMadeleine) [inline docs/api/customers.rst]: Customer's

File: docs/api/customers.rst [hunk 5]
```diff
@@ -60,6 +60,195 @@ If you request for a more detailed data, you will receive an object with followi
...
++--------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| birthday                 | request        | *(optional)* Customers birthday                                                                      |
```
Comment (TheMadeleine) [inline docs/api/customers.rst]: Customer's

File: docs/api/customers.rst [hunk 6]
```diff
@@ -60,6 +60,195 @@ If you request for a more detailed data, you will receive an object with followi
...
+        "emailCanonical":"john.diggle@yahoo.com",
+        "firstName":"John",
+        "lastName":"Diggle",
+        "gender":"m",
+        "group":[
+
+        ]
+    }
+
+If you try to create a customer without email or gender, you will receive a 400 error.
```
Comment (TheMadeleine) [inline docs/api/customers.rst]: ``` a ``400 Bad Request`` error ```

File: docs/api/customers.rst [hunk 7]
```diff
@@ -60,6 +60,195 @@ If you request for a more detailed data, you will receive an object with followi
...
+.. code-block:: text
+
+    GET /api/v1/customers/{id}
+
++---------------+----------------+-------------------------------------------------------------------+
+| Parameter     | Parameter type | Description                                                       |
++===============+================+===================================================================+
+| Authorization | header         | Token received during authentication                              |
++---------------+----------------+-------------------------------------------------------------------+
+| id            | url attribute  | Id of requested resource                                          |
```
Comment (TheMadeleine) [inline docs/api/customers.rst]: the requested resource

**Comment by pjedrzejewski** [comment]
Thanks Magda!

**Comment by pjedrzejewski** [comment]
OMG, I did it again, sorry! :) Thank you Ania! :)

**Pull request merged**